docs/howto/cron.rst

```diff
@@ -23,20 +23,41 @@ in time, and the database is responsible for making sure deferring only happens
 period, even with multiple workers. This means that you can have high availability
 periodic scheduling: as long as at least one worker is up, the tasks will be deferred.
 
-.. warning::
-
-    The workers will defer the task at the requested period, but the corresponding
-    jobs may be executed later, depending on how busy the workers are.
-
 The cron syntax is on 5 columns (minute, hour, day, month, day of week). Here, an
 optional 6th column is supported for seconds with the same syntax, allowing you to make
 a periodic task as frequent as "1 per second".
 
+Caveats
+-------
+
+When using periodic tasks there are a few things to know:
+
+- Tasks are periodically **deferred** but their execution time depends on the state of
+  your queue. If your workers are overwhelmed with long tasks, periodic executions could
+  be delayed. One possible solution involves using queues to make sure short tasks are
+  not delayed by long tasks.
+- Workers are responsible for deferring periodic tasks. If not a single worker is
+  running, then periodic tasks will not get scheduled. On the other hand, even if you
+  have multiple workers, periodic tasks will only be deferred once per period.
+- If a worker is occupied by a long-standing synchronous task, it will not be available
+  to defer periodic tasks. If you use sync tasks, ensure that at least one of your
+  workers is executing tasks that are usually shorter to run than your shortest periodic
```

Comment:
```suggestion
  workers executes tasks that are usually shorter to run than your shortest periodic
```

Reply:
"is assigned"

Reply:
Woops, forgot to push the commit that fixed this.